src/main/java/seedu/address/logic/commands/MarkCommand.java

```diff
@@ -26,62 +27,75 @@ public class MarkCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 "
             + "tut/1";
 
-    public static final String MESSAGE_ARGUMENTS = "Index: %1$d, Tutorial: %2$s";
-
-    public static final String MESSAGE_MARK_SUCCESS = "Marked present in Tutorial: %2$s for Person: %1$s";
-    public static final String MESSAGE_MARK_UNNECESSARY =
-            "Person: %1$s is already marked as present for Tutorial: %2$s";
-
-    private final Index index;
+    private final List<Index> indexList = new ArrayList<>();
```

Comment:
IndexList may not be necessary here since it is either 1 index or nothing. Likewise, shouldMarkAll could also be done by just checking the index.

Reply:
yes ser